src/HotChocolate/Fusion/src/Core/Planning/Steps/SelectionPath.cs

```diff
@@ -35,8 +35,8 @@ public override string ToString()
 
         while (current is not null)
         {
-            sb.Append('/');
-            sb.Append(Selection.ResponseName);
+            sb.Insert(0, '/');
+            sb.Insert(1, current.Selection.ResponseName);
```

Comment:
This is purely for debugging purposes. Previously the ToString would return something like `name/brand/productById` instead of `productById/brand/name` which would be more helpful to understand the path.